packages/@aws-cdk/aws-appmesh/lib/gateway-route.ts

```diff
@@ -26,6 +27,72 @@ export interface IGatewayRoute extends cdk.IResource {
    * The VirtualGateway the GatewayRoute belongs to
    */
   readonly virtualGateway: IVirtualGateway;
+
+  /**
+   * Grants the given entity all read permissions for this GatewayRoute.
+   */
+  grantRead(identity: iam.IGrantable): iam.Grant;
+
+  /**
+   * Grants the given entity all write permissions for this GatewayRoute.
+   */
+  grantWrite(identity: iam.IGrantable): iam.Grant;
+}
+
+abstract class GatewayRouteBase extends cdk.Resource implements IGatewayRoute {
+  /**
+   * The name of the GatewayRoute
+   *
+   * @attribute
+   */
```

Comment:
If the JSDocs are the same as the implemented property / method, there's no reason to repeat them for subclasses.

Same comment everywhere below.